cmd/reports-controller/main.go

```diff
@@ -271,6 +274,7 @@ func main() {
 		setup.KubeClient,
 		setup.KyvernoClient,
 		setup.RegistrySecretLister,
+		apicall.NewAPICallConfiguration(maxAPICallResponseLength),
```

Comment:
Do we need this in 1.10? If so, is the default size limit enough for the final report?

Reply:
Yes, imo, we should add this to 1.10 as well (I will create a custom PR for release1.10 once we have finalised this change).

The max size value looks good to me, @eddycharly, can you please take a look.

Reply:
Why does the report size matter here?

Anyways, the etcd object size limit is 1.5MB.

Reply:
I was wondering if policy report data is stored in the annotation for last applied changes?